### ISSUE TITLE
How to continue training with new settings?

### ISSUE THREAD

**Issue by peastman** [torchmd/torchmd-net#86]
After training a model, is there any way to do a new training run, starting from the result of the previous run?  I know I can use the `load_model` option, but that resumes an existing run from a checkpoint with exactly the same settings used before.  Here are the sorts of things I'd like to be able to do:

- Train on one dataset, then fine tune on another dataset.
- Initially train just on energies, then fine tune with forces as well.
- Train in reduced precision (half or TF32), then fine tune in full precision.

**Comment by PhilippThoelke** [comment]
If you load a checkpoint using the `--load-model` arg you can simply also specify a config file through `--config` or add command line arguments directly, which will overwrite the args from the checkpoint file. Of course adapting architectural hyperparameters will break the training as loading the checkpoint's state dict wouldn't work. The arguments you listed should work perfectly fine though.

**Comment by peastman** [comment]
That's what I had expected, but I can't get it to work.  For example, a training run stopped earlier than I wanted it to.  So I increased `early_stopping_patience` and decreased `lr_min` and tried to resume.  But it didn't make a difference.  It still stopped training after just a few epochs, at the same point it had on the original training run.

**Comment by peastman** [comment]
Here is another example.  I just trained a model on energies.  Now I'm trying to fine tune it including forces as well.  When I tell it to load from the checkpoint, it ignores the learning rate in the configuration file and instead starts from the final learning rate at the end of the original training run.

**Comment by PhilippThoelke** [comment]
Ah right, pytorch-lightning stores information about optimizer state, current lr and so on in the checkpoint. I just added a new argument to `train.py` to reset the trainer state when loading a model. You can simply add `reset_trainer: true` to your config file. Let me know if this resolves it.

**Comment by peastman** [comment]
Thanks!  That seems to work.
